search-commons/src/main/java/no/unit/nva/search/resource/ResourceAccessFilter.java

```diff
@@ -123,25 +124,35 @@ private static URI getCurationInstitutionId(RequestInfo requestInfo)
                 : requestInfo.getPersonAffiliation();
     }
 
-    private QueryBuilder getEditorFilterWith(String institutionId) {
+    private QueryBuilder getContributingOrganisationAccessFilter(String institutionId) {
         return QueryBuilders.termQuery(CONTRIBUTOR_ORG_KEYWORD, institutionId)
                 .queryName(EDITOR_FILTER);
     }
 
-    private QueryBuilder getCuratorFilterWith(String institutionId) {
+    private QueryBuilder getCuratingInstitutionAccessFilter(String institutionId) {
         return QueryBuilders.termQuery(CURATING_INST_KEYWORD, institutionId)
                 .queryName(CURATOR_FILTER);
     }
 
+    /**
+     * Only Editors are allowed to see UNPUBLISHED publications
+     *
+     * @param publicationStatus status to check
+     * @return true if allowed
+     */
+    private boolean statusAllowed(PublicationStatus publicationStatus) {
+        return isEditor() || publicationStatus != UNPUBLISHED;
+    }
+
+    private boolean isCuratorWantsNoFilter() {
```

Comment:
This name is a bit confusing, but I don't have a better suggestion

Reply:
business rules, hand made ftw